resilience4j-core/src/main/java/io/github/resilience4j/core/Registry.java

```diff
@@ -75,6 +76,11 @@
      */
     C getDefaultConfig();
 
+    /**
+     * @return global configured registry tags
+     */
+    Map<String, String> getRegistryTags();
```

Comment:
I would prefer to call it `getTags` as well

Reply:
done